packages/test/test-end-to-end-tests/src/test/sharedInterval.spec.ts

```diff
@@ -371,8 +371,8 @@ describeCompat("SharedInterval", "NoCompat", (getTestObjectProvider, apis) => {
 			intervals.add({ start: 0, end: 2 });
 			assertIntervals([{ start: 0, end: 2 }]);
 
-			for (let j = 0; j < 10; j++) {
```

Comment:
Why is this needed?

Reply:
I reduced the number of ops this test sends since it was the only one sending more than 200 unsummarized ops (a setting I changed in local server to enable the test I added)